### PR TITLE
Refactoring of gsl.copy_properties.sh

### PR DESCRIPTION
**Pull request by ldz1**
- Remove unused function inside copy_properties.
- Remove unneeded nests in the output file.

### CODE REVIEW

**Comment by ldz1** [comment]
Windows build probably corrupted. See https://github.com/libbitcoin/libbitcoin-build/issues/218

Similar changes for cmd are needed. It's coming soon.

File: templates/shared/common_msvc_artifacts.gsl [hunk 1]
```diff
@@ -94,7 +94,7 @@ function global.get_dependencies_nuget(result, repositories, configure,\
         _dependency, my.compiler) & matches_option(_dependency, my.option)
 
         # embedded prefix
-        define my.name = "lib$(_dependency.name)"
```
Comment (evoskuil) [inline templates/shared/common_msvc_artifacts.gsl]: Dependency names are conceptual, not prefixed with "lib".

**Comment by ldz1** [comment]
My current PR is wrong. Walking the dependency tree has to be recursive

**Comment by ldz1** [comment]
(I leave the TODO open so that the PR can be completed first.)

**Pull request merged**